scripts/build/dependencies-npm.sh

```diff
@@ -94,15 +94,16 @@ fi
 function run_install() {
   npm install $INSTALL_OPTS
 
-  # Turns out that if `npm-shrinkwrap.json` contains development
-  # dependencies then `npm install --production` will also install
-  # those, despite knowing, based on `package.json`, that they are
-  # really development dependencies. As a workaround, we manually
-  # delete the development dependencies using `npm prune`.
   if [ "$ARGV_PRODUCTION" == "true" ]; then
+
+    # Turns out that if `npm-shrinkwrap.json` contains development
+    # dependencies then `npm install --production` will also install
```

Comment:
Does this comment imply that when `"$ARGV_PRODUCTION" == "true"` we ought to be adding `--production` to `$INSTALL_OPTS`?

(I guess there's a slim chance that `npm install --production` might get fixed in future, and in that case `npm install --production; npm prune --production` will probably be quicker than `npm install; npm prune --production`?)